code-challenges.js

```diff
@@ -10,45 +10,154 @@
+// console.log(myArray)
+/////////////////////////////////////////////////////////////////////// Todd's Pseudocode //////////////////////////////////////////////////////////////
+// All I have to say is you do the best You can do with what you have.
+//I took what I had learned during the week and stuck with what I know I understand that it's probably a long drawn out process but I got the job done.
+//I knew that by using the slice method I would be able to chop a single number out of an array so that's what I did I chopped it 10 times for 10 numbers.
+//Then I figured out which number went where and used the concat method ten times to make a new variable called myArray.
+// After I had done all of this I wanted to know if there was a simpler way so I got online and talked to a couple of other coders that I know and this was the solution they gave me.
+// 
+// var myNewNumber = myNumbers1.concat(myNumbers2)
+// // console.log(myNewNumber)
+
+// var myNewArray = myNewNumber.sort(function(a,b){ return a - b});
+// console.log(myNewArray)
+//This is a function that allows you to take an array break it down by numbers. 
+// sort that array then return the assorted numbers.
+//Reading the syllabus we should be learning about these types of functions next week
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 // --------------------3) Create a function that takes in a string of a single word and returns the string with all letters reversed.
 // Use the test variables provided below. Expected output: "ovarb", "eilrahc"
 
-var myString1 = "bravo"
-var myString2 = "charlie"
-
-
+// var myString1 = "bravo"
+// var myString2 = "charlie"
+
+// var newString3 = myString1.split("")
+// var newString4 = newString3.reverse()
+// var newString5 = myString2.split("")
+// var newString6 = newString5.reverse()
+// var newString7 = newString4.join("")
+// var newString8 = newString6.join("")
+// console.log(newString7, newString8)
+/////////////////////////////////////////////////////////////////////// Todd's Pseudocode //////////////////////////////////////////////////////////////
+// First thing I had to do was convert the strings into an array because the .reverse is an a ray method.
+// As you can see I worked with one and then the other just to make things simple for me.
+// Once I got them into a race I then reversed them and then using the .join Method I was then able to convert them back into a string.
+// Then I worked on the second bit of code for Charlie once I knew that I had bravo correct.
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Your logic here is sound, just remember the prompt calls for a function. Containing your logic in a function allows us to package up our logic and reuse it whenever the situation fits the case.